src/redux/actions/auth/authActions.js

```diff
@@ -19,10 +19,10 @@ const loginAction = ({ username, password }) => (dispatch) => {
     localStorage.setItem('username', `${username}`);
     return dispatch(login(response));
   }).catch((error) => {
-    if (error.response) {
-      return dispatch(loginFailed(error.response.data.detail));
+    if (error.response === undefined) {
+      return dispatch(loginFailed('Network Error. Please try again later!'));
```

Comment:
Can you use a constant for this error message?

Reply:
done